git/attribs.go

```diff
@@ -58,6 +58,12 @@ func GetAttributePaths(workingDir, gitDir string) []AttributePath {
 		for scanner.Scan() {
 			line := scanner.Text()
 
+			line := strings.TrimSpace(line)
+
+			if strings.HasPrefix("#") {
+				continue
```

Comment:
The [tests](https://ci.appveyor.com/project/lfs-build/git-lfs/build/1.0.2361) show a compilation error here:

```
git\attribs.go:63: not enough arguments in call to strings.HasPrefix
	have (string)
	want (string, string)
```

The [docs](https://golang.org/pkg/strings/#HasPrefix) show a function signature of: `strings.HasPrefix(s, prefix string) bool`. In this case, `line` is the string, and `"#"` is the prefix you're checking for:

```go
if strings.HasPrefix(line, "#") {
```